pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pygls"
-version = "1.3.1"
+version = "2.0.0a1"
```

Comment:
@tombh do we edit the version number manually, or does the release automation do that?

Reply:
We edit it manually. I'm surprised I hadn't made a `RELEASING.md` doc, so I've made a PR that outlines the release process: https://github.com/openlawlibrary/pygls/pull/488